.github/workflows/main.yml

```diff
@@ -25,35 +27,45 @@ jobs:
           - 5432:5432
 
     steps:
-    - name: System dependencies
-      run: |
-        sudo apt-get update
-        sudo apt-get install gdal-bin
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.x
-      uses: actions/setup-python@v2
-      with:
-        # Semantic version range syntax or exact version of a Python version
-        python-version: '3.x'
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r tests/requirements.txt
-        pip install -e .
-    - name: Lint with flake8
-      run: |
-         flake8 ./councilmatic_core/*.py
-    - name: Test with pytest
-      run: |
-        pytest -sv
-    - name: Build and publish
-      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
-      env:
-        TWINE_USERNAME: __token__
-        TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
-      run: |
-        pip install twine wheel
-        pip wheel -w dist --no-deps .
-        python setup.py sdist
-        twine upload dist/*
-      continue-on-error: true
+      - name: System dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install gdal-bin libreoffice unoconv nodejs
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v2
+        with:
+          # Semantic version range syntax or exact version of a Python version
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r tests/requirements.txt
+          pip install -e .
+          npm install --legacy-peer-deps
+      - name: Test with pytest
+        run: |
+          flake8 councilmatic_core tests
+          npx eslint councilmatic_core/static/js/*.js
+          pytest -sxv
+
```

Comment:
let's also run a black test?

Reply:
to maek sure it is formatted acorrding to black.

Reply:
Done!